data/data.json

```diff
@@ -4081,10 +4089,28 @@
       "section": "feedback",
       "key": "page_header",
       "English": "Help us improve",
-      "French": "(fra) Help us improve",
+      "French": "Aidez-nous à nous améliorer",
       "id": "feedback.page_header",
       "Used by code": [],
       "Safe to Delete?": []
+    },
+    {
+      "section": "feedback",
+      "key": "submitted",
+      "English": "Thank you for taking the time to give us feedback. We are listening and constantly improving.",
+      "French": "Merci d’avoir pris le temps de nous faire part de vos commentaires. Nous sommes à l’écoute et nous nous améliorons constamment.",
+      "id": "feedback.submitted",
+      "Used by code": [],
+      "Safe to Delete?": []
```

Comment:
We can probably delete the "Used by code" and "Safe to Delete" columns in translations in Airtable.

Reply:
But is it safe to delete the "Safe to Delete" column? 🤔